ca.uwaterloo.Raven.json

```diff
@@ -0,0 +1,82 @@
+{
+  "app-id": "ca.uwaterloo.Raven",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "Raven.exe",
+  "finish-args":
+  [
+    "--filesystem=home"
+  ],
+  "cleanup": 
+  [
+    "/lib/pkgconfig",
+    "/lib/*.la",
+    "/lib/*.a",
+    "/lib/*.settings",
+    "/include",
+    "/share"
```

Comment:
this cleanup rule remove the important files likes icon, appstream and .desktop

In general this rule is wrong.

Reply:
this is the cause of the build failure.

Reply:
Ah makes sense! Thank you! I have removed the "/share" from the cleanup commands.

Reply:
however the cleanup rule can be local to a module... and only apply to files installed by said module.